.github/workflows/ci-latest-release.yml

```diff
@@ -65,11 +71,19 @@ jobs:
       - name: Get release tag
         id: vars
         run: |
-          if [ ${{ github.ref }} == "refs/heads/main" ]; then
-            echo "tag=latest" >> $GITHUB_OUTPUT
+          if [[ ${{ github.event_name }} == "workflow_dispatch" ]]; then
+            # checkout branch but use goreleaser config from latest
+            echo "Checking out tag: ${{ inputs.tag }}"
+            git checkout ${{ inputs.tag }}
```

Comment:
let's merge  https://github.com/kubearmor/KubeArmor/pull/2026 and then we can rebase this.